requirements.txt

```diff
@@ -2,7 +2,7 @@ numpy~=1.23
 scipy~=1.8
 cvxpy~=1.2
 cvxopt~=1.3.0
-cachetools~=5.0.0
+cachetools~=5.3.0
```

Comment:
```suggestion
cachetools~=5.3.0 # this should be at least v5.3.0 because qualtran v0.6 (as of 2025/01/14) requires it, no known reason for why it cannot be anything higher.
```